UltiSnips/django.snippets

```diff
@@ -1,238 +1,367 @@
 priority -50
 
-# Generic Tags
-snippet %
-{% ${1} %}${2}
+# This files will define django snippets from sublime text djaneiro
+
+###########################################################################
+#                             FORMS SNIPPETS                              #
+###########################################################################
+snippet form "Form"
```

Comment:
Do you have examples of this? any one I could know exactly how use it... At first I was interested on it but I couldn't find examples.